ci-operator/step-registry/workers-rhel/vsphere-provision/workers-rhel-vsphere-provision-commands.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+KUBECONFIG=${SHARED_DIR}/kubeconfig
+SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+SSH_USER="test"
```

Comment:
How about getting it from `${SHARE_DIR}` ?

Reply:
Do you mean get from ${SHARE_DIR}/bastion_ssh_user? The ref ipi-conifg-vsphere-bastionhost is only needed when scaleup rhel worker in a disconnected network? I plan to add it into Vault, wdyt?

Reply:
Do you mean not each rhel scaleup case has `${SHARE_DIR}/bastion_ssh_user`, e.g: connected clusters on vsphere, right?

Reply:
Okay. I got the point, the `SSH_USER` is not for bastion host, but rhel worker login.

Reply:
If that, I prefer make the rhel image and the rhel ssh login to become ENV var for this ref, so that someone can easily overwrite them when needed.

Reply:
ok for me, will update.

Reply:
Per our experience in our QE's rhel scaleup auto, for rhel image, version, and rhel ssh login, should be bound together.